spec/index.bs

```diff
@@ -10776,7 +10778,7 @@ It must only be included by interfaces which also include those mixins.
             - |pipeline| must not be `null`.
             - All bind groups used by the pipeline must be set and compatible with the pipeline layout:
                 For each pair of ({{GPUIndex32}} |index|, {{GPUBindGroupLayout}} |bindGroupLayout|) in
-                |pipeline|.{{GPUPipelineBase/[[layout]]}}.{{GPUPipelineLayout/[[bindGroupLayouts]]}}:
+                non-`null` |pipeline|.{{GPUPipelineBase/[[layout]]}}.{{GPUPipelineLayout/[[bindGroupLayouts]]}}:
```

Comment:
Non-null seems associated with the pipeline layout here instead of each checking each BGL independently.

Reply:
Done.